src/components/PostList/PostList.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Post from '../Post';
+
+const PostList = ({posts}) => (
+  <div>
+    {posts.map(post => <Post post={post} key={post.id} />)}
```

Comment:
```suggestion
    {posts.map(onePost => <Post postData={onePost} key={post.id} />)}
```
Это как пример, потому что три слова post подряд выглядит не очень читаемо